packages/interbit/src/tests/integrationTests/helpers.js

```diff
@@ -0,0 +1,41 @@
+const log = require('../../log')
+
+const waitForState = (chainInterface, predicate = () => {}, timeout = 2000) =>
+  new Promise((resolve, reject) => {
+    let unsubscribe = () => {}
+    unsubscribe = chainInterface.subscribe(() => {
+      const state = chainInterface.getState()
+      if (predicate(state)) {
+        unsubscribe()
+        resolve()
+      }
+    })
+
+    setTimeout(() => {
+      unsubscribe()
+      reject(new Error(`Waiting for block timed out after ${timeout}ms`))
+    }, timeout)
+  })
+
+const dispatchAction = async (chainInterface, action) => {
+  log.action(action)
+  await chainInterface.dispatch(action)
+}
+
+const getChainId = (alias, chains) => chains[alias].chainId || chains[alias]
```

Comment:
I did a double-take on this one. I assume this is because chains[alias] can be either an object or a string.

Reply:
Correct

This is a selector that may run on the dev mode chains list or build manifest, which both have a different form. It allows a uniform assertChainsConfigured* function that can assert on the config/covenant provided for start and deploy modes.